src/group_theory/abelianization.lean

```diff
@@ -106,4 +111,61 @@ theorem hom_ext (φ ψ : abelianization G →* A)
   (h : φ.comp of = ψ.comp of) : φ = ψ :=
 monoid_hom.ext $ λ x, quotient_group.induction_on x $ monoid_hom.congr_fun h
 
+section map
+
+variables {H : Type v} [group H] (f : G →* H)
+
+/-- The map operation of the `abelianization` functor -/
+def map : abelianization G →* abelianization H := lift (of.comp f)
+
+lemma map_apply {x : abelianization G} : map f x = lift (of.comp f) x := rfl
```

Comment:
This lemma is pretty pointless now, if you want to unfold `map` you can just use `rw map`.